config/manager/kustomization.yaml

```diff
@@ -1,2 +1,8 @@
 resources:
 - manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: ghcr.io/jacob.aronoff/opentelemetry-operator/opentelemetry-operator
+  newTag: e2e
```

Comment:
I assume that was not intended? :x: 

cc @jaronoff97

Reply:
yeah 😓 i fixed it in #2218